src/interfaces/IWorkValidation.sol

```diff
@@ -1,9 +1,10 @@
 // SPDX-License-Identifier: Apache-2.0
 pragma solidity ^0.8.0;
 
-abstract contract IWorkValidation {
-    function validateWork(uint256 domainId, uint256 jobId, address provider, uint256 nodeId, bytes calldata data)
+interface IWorkValidation {
```

Comment:
We need some method to get work to be validated. I think the current setup with the SyntheticDataWorkValidator impl is ok since we have a central validator for now.

So one way I think this works is that the central validator queries for `getWorkSince` to know what work hasnt been validated yet and schedules them out to leviticus workers. The results are then used to `invalidateWork` if we find a piece of work to be wrong.

Reply:
But with multiple validation miners, its not clear to me how the miners can coordinate to split the work here. Maybe @JannikSt, you and I should have a sync about this?